yjit/src/stats.rs

```diff
@@ -141,6 +141,19 @@ macro_rules! make_counters {
     }
 }
 
+/// Macro to increase a counter by name and count
+macro_rules! add_counter {
+    // Unsafe is ok here because options are initialized
+    // once before any Ruby code executes
+    ($counter_name:ident, $count:expr) => {
+        #[allow(unused_unsafe)]
+        {
+            unsafe { $crate::stats::COUNTERS.$counter_name += $count as u64 }
+        }
+    };
+}
+pub(crate) use add_counter;
+
```

Comment:
```suggestion
```
seems like this was added by accident?

Reply:
What do you mean? Do we have other ways to increase a counter by a non-1 amount?

Reply:
Oh sorry, I failed to see the call site.